src/tests/unit/tests/views/content/content-link.test.tsx

```diff
@@ -30,60 +36,58 @@ describe('ContentLink', () => {
     };
 
     it('render null when reference is not defined', () => {
-        const result = shallow(<ContentLink deps={deps} reference={null} />);
-        expect(result.debug()).toMatchSnapshot();
+        const renderResult = render(<ContentLink deps={deps} reference={null} />);
+        expect(renderResult.asFragment()).toMatchSnapshot();
     });
 
     it('renders from content, only have the icon', () => {
-        const result = shallow(
+        const renderResult = render(
             <ContentLink deps={deps} reference={content.for.testing} iconName={'test icon 1'} />,
         );
-        expect(result.debug()).toMatchSnapshot();
+        expect(renderResult.asFragment()).toMatchSnapshot();
     });
 
     it('renders from path, only have the icon', () => {
-        const result = shallow(
+        const renderResult = render(
             <ContentLink deps={deps} reference={contentPath} iconName={'test icon 2'} />,
         );
-        expect(result.debug()).toMatchSnapshot();
+        expect(renderResult.asFragment()).toMatchSnapshot();
     });
 
     it('renders with only text', () => {
-        const result = shallow(
+        const renderResult = render(
             <ContentLink deps={deps} reference={contentPath} linkText={'test'} />,
         );
-        expect(result.debug()).toMatchSnapshot();
+        expect(renderResult.asFragment()).toMatchSnapshot();
     });
 
     it('renders with both text and icon', () => {
-        const result = shallow(
+        const renderResult = render(
             <ContentLink
                 deps={deps}
                 reference={contentPath}
                 linkText={'test'}
                 iconName="test icon"
             />,
         );
-        expect(result.debug()).toMatchSnapshot();
+        expect(renderResult.asFragment()).toMatchSnapshot();
     });
 
     it('renders without tooltip', () => {
-        const result = shallow(
+        const renderResult = render(
             <ContentLink
                 deps={deps}
                 reference={contentPath}
                 linkText={'test'}
                 hideTooltip={true}
             />,
         );
-        expect(result.debug()).toMatchSnapshot();
+        expect(renderResult.asFragment()).toMatchSnapshot();
     });
 
-    it('reacts to a click', () => {
-        const result = shallow(<ContentLink deps={deps} reference={contentPath} />);
-
-        result.find(NewTabLinkWithTooltip).simulate('click');
-
-        expect(openContentPage).toBeCalledWith(undefined, contentPath);
+    it('reacts to a click', async () => {
+        render(<ContentLink deps={deps} reference={contentPath} />);
+        await getMockComponentClassPropsForCall(NewTabLinkWithTooltip).onClick();
+        expect(openContentPage).toHaveBeenCalledTimes(1);
```

Comment:
Check once with Vikash on this change

Reply:
We can revert this change back to original.

Reply:
Yes, reverted it back to original.